jbehave-support-core/src/main/java/org/jbehavesupport/core/internal/web/WebScreenshotCreator.java

```diff
@@ -25,24 +25,34 @@
 @RequiredArgsConstructor
 public class WebScreenshotCreator {
 
+    public enum type {FAILED, DEBUG, MANUAL, WAIT, STEP}
```

Comment:
One more thing I noticed we seem to be mixing the word fail and error a lot which at least for me causes some confusion - I would vote to unify it to just one term (maybe ERROR because we have error_screenshots already? but no strong feelings either way)

Reply:
I just stick to existing solutions for FailScreenshotsReporterExtension (as I am overwriting it).